docusaurus/src/css/custom.css

```diff
@@ -0,0 +1,104 @@
+/* stylelint-disable unit-blacklist, media-feature-name-value-whitelist */
```

Comment:
We recently upgraded to the most recent version of docusaurus. Can you please do it as well? Please note that this file has changed and must be copied over again.

Reply:
Updated ✅